repo.go

```diff
@@ -110,6 +111,24 @@ func parseRepo(defaultSchema, defaultHost, defaultOrganization, name string) (re
 	return r, nil
 }
 
+func (r repo) isGithubRemote() (ok bool) {
+	cmd := xexec.Fmt("git remote -v")
+	out, err := cmd.Output()
```

Comment:
Use [`cmd.CombinedOutput`](https://golang.org/pkg/os/exec/#Cmd.CombinedOutput) so we can use this in the error as well.